systest/license/license_test.go

```diff
@@ -93,95 +90,108 @@ type responseStruct struct {
 	License map[string]interface{} `json:"license"`
 }
 
-func TestEnterpriseLicense(t *testing.T) {
-	enterpriseLicenseURL := "http://" + testutil.SockAddrZeroHttp + "/enterpriseLicense"
-
-	var tests = []struct {
-		name       string
-		licenseKey []byte
-		code       string
-		user       string
-		message    string
-	}{
-		{
-			"Using expired entrerprise license key, should be able to extract user information",
-			expiredKey,
-			`Success`,
-			`Dgraph Test Key`,
-			``,
-		},
-		{
-			"Using invalid entrerprise license key should return an error",
-			invalidKey,
-			``,
-			``,
-			`while extracting enterprise details from the license: while reading PGP message from license file: openpgp: unsupported feature: public key version`, //nolint:lll
-		},
-		{
-			"Using empty entrerprise license key should return an error",
-			[]byte(``),
-			``,
-			``,
-			`while extracting enterprise details from the license: while decoding license file: EOF`,
-		},
-	}
+type TestInp struct {
+	name       string
+	licenseKey []byte
+	code       string
+	user       string
+	message    string
+}
+
+var tests = []TestInp{
```

Comment:
see if you could avoid declaring a named type here too

Reply:
I am using the name TestInp in the suite structure so, cannot avoid declaring the name.